download/download_test.go

```diff
@@ -0,0 +1,99 @@
+package download
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestSuite struct {
+	suite.Suite
+}
+
+func TestConfigTestSuite(t *testing.T) {
+	suite.Run(t, new(TestSuite))
+}
+
+func (suite *TestSuite) SetupTest() {
+
+}
+
+func (suite *TestSuite) TestNoArgument() {
+
+	os.Args = []string{"download"}
+
+	err := Download(os.Args)
+	assert.EqualError(suite.T(), err, "failed to find location of files, no argument passed")
+}
+
+func (suite *TestSuite) TestdownloadListFileWrongUrl() {
+
+	url := "someUrl"
+	filePath := "."
+	err := downloadListFile(url, filePath)
+
+	assert.EqualError(suite.T(), err, "failed to download file, reason: Get \"someUrl\": unsupported protocol scheme \"\"")
+}
+
+func (suite *TestSuite) TestWrongUrlsFile() {
+
+	urlsListPath, err := ioutil.TempFile(os.TempDir(), "urls_list-")
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
```suggestion
	assert.NoError(suite.T(), err)
```

the log Fatal will kill the process making the defer on the next lint to not run.